examples/port-2-alibaba2-connection/terraform.tfvars.example

```diff
@@ -0,0 +1,35 @@
+equinix_client_id      = "MyEquinixClientId"
+equinix_client_secret  = "MyEquinixSecret"
+
+connection_name             = "Port_2_alibaba"
+connection_type             = "EVPL_VC"
+notifications_type          = "ALL"
+notifications_emails        = ["example@equinix.com"]
+bandwidth                   = 50
+purchase_order_number       = "1-323292"
+aside_port_name             = "sit-tb1-dc-e5.tlab,10GSMF,A,001,201257, 21951980"
+aside_vlan_tag              = 2019
+zside_ap_type               = "SP"
+zside_ap_authentication_key = "<Alibaba Auth Key>"
+zside_ap_profile_type       = "L2_PROFILE"
+zside_location              = "SY"
+zside_sp_name               = "Alibaba Cloud Express Connect"
+zside_seller_region         = "ap-southeast-2"
+zside_ap_profile_type       = "L2_PROFILE"
+zside_ap_type               = "SP"
+zside_location              = "SV"
+zside_seller_region         = "us-west-1"
+zside_sp_name               = "Alibaba Cloud Express Connect"
+region                      = "us-west-1"
+local_gateway_ip            = "10.0.0.1"
+peer_gateway_ip             = "10.0.0.2"
+peering_subnet_mask         = "255.255.255.252"
+virtual_border_router_name  = "Port_2_Alibaba_VBR"
+min_rx_interval             = 1000
+min_tx_interval             = 1000
+detect_multiplier           = 10
+access_key                  = "<Alibaba_Cloud_Access_Key>"
+secret_key                  = "<Alibaba_Cloud_Secret_Key>"
+project_id                  = "<Fabric_Project_ID>"
+physical_connection_id      = "<Physical_Connection_ID>"
```

Comment:
This is computed. Remove.

Reply:
Addressed with commit: [e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb](https://github.com/equinix/terraform-equinix-fabric/pull/155/commits/e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb)